packages/caliper-vscode-extension/artifacts/json-schema/hyperledger-caliper-runtime-configuration.schema.json

```diff
@@ -0,0 +1,543 @@
+{
+    "$schema": "https://json-schema.org/draft-06/schema",
+    "$ref": "#/definitions/ConfigurationHelper",
+    "definitions": {   
+        "ConfigurationHelper": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "caliper": {
```

Comment:
Every root/middle/leaf property must have a user-friendly description, summarizing the property(group) as precisely as possible